code/src/index.css

```diff
@@ -1,13 +1,183 @@
+* {
+	margin: 0;
+	padding: 0;
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	margin: 20px;
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+	background-color: #181818;
+	cursor: default;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+	font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
+}
+
+header {
+	width: 300px;
+	margin: 0 auto;
+}
+
+.heading {
+	font-family: Helvetica;
+	font-size: 18px;
+	color: #ffffff;
+}
+
+hr {
+	opacity: 0.1;
+}
+
+.album-container {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
+	column-gap: 20px;
+	row-gap: 45px;
+	padding-top: 20px;
+}
+
+.album-image-container {
+	position: relative;
+}
+
+.album-image {
+	width: 300px;
+	display: block;
+	margin-bottom: 10px;
+}
+
+.album-cover-overlay {
+	opacity: 0;
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	padding: 112.5px 42.5px;
```

Comment:
If we understand it correctly, you have added this very specific padding to center the icons on the album. Instead you could have used the flex attributes to align and center it on the card.